qemu/tests/cfg/vsock.cfg

```diff
@@ -24,3 +24,11 @@
               devno = 'fe.1.2345'
         - migration_with_vsock:
             type = migration_with_vsock
+        - iommu_enabled:
+            no RHEL.6 RHEL.7 RHEL.8.0 RHEL.8.1 RHEL.8.2 RHEL.8.3 RHEL.8.4
+            required_qemu = [5.2.0,)
+            #refer to bug 1879052,1894101
+            virtio_dev_iommu_platform = on
+            ppc64, ppc64le:
```

Comment:
`pc` machine should also apply these two parameters if I understand correctly.

Reply:
@qcheng-redhat hello, I checked the existing tp-qemu case and found that iommu tests are only on q35.
So this test is for x86: `only q35` or `pc,  ppc64, ppc64le:`
Could you confirm this?

Reply:
For x86, only q35.

Reply:
thanks all, Corrected.